### DISCUSSION TITLE
bt error

### DISCUSSION THREAD

**Discussion by perromon28** [espressif/arduino-esp32#9166] [Q&A]
hello, i'm trying to have the mac bluetooth of esp32 for a rc robot, but i find this problem: `C:\Users\HP\AppData\Local\Arduino15\packages\esp32\hardware\esp32\2.0.11/tools/sdk/esp32/include/bt/host/bluedroid/api/include/api/esp_gap_bt_api.h:500:11: note: declared here
 esp_err_t esp_bt_gap_set_scan_mode(esp_bt_connection_mode_t c_mode, esp_bt_discovery_mode_t d_mode);`
what can i do?

**Comment by P-R-O-C-H-Y** [comment]
Hi @perromon28, can you please post your sketch?
